src/postgres.c

```diff
@@ -72,6 +75,47 @@ _cpycmd(char *host, char *generation)
     return cpycmd;
 }
 
+void
+_commit(Meta *m)
+{
+    PQputCopyEnd((*m)->conn_master, NULL);
+    if ((*m)->conninfo_replica)
+        PQputCopyEnd((*m)->conn_replica, NULL);
+
+    (*m)->count = 0;
+    (*m)->copy  = 0;
+    (*m)->commit_iter = 0;
+}
+
+void *
+_commit_worker(void *meta)
+{
+    Meta *m = (Meta *) meta;
+
+    #if __linux__
```

Comment:
Nitpick:
This will break on linux[<2.6.9] and freebsd has the same interface available so I think best is to do `#ifdef PR_SET_NAME` so when it matters we won't have to care.